### DISCUSSION TITLE
Bonzeb help

### DISCUSSION THREAD

**Discussion by Zebfin** [bonsai-rx/bonsai#1625] [General]
Hi I am looking for help in bonzeb. People who have used and willing to guide pls help.

**Comment by bruno-f-cruz** [comment]
Hi 
Please read the community guidelines before posting. It is very important to follow these guidelines as it ensures that future users can use this resource. thanks

**Comment by Zebfin** [comment]
Dear Sir

Can you please help. I need help in putting bonzeb can you recommend a
person.

Thanks

Rajat

**Comment by glopesdev** [comment]
@Zebfin the documentation for BonZeb can be found [online](https://ncguilbeault.github.io/BonZeb/). If you would like help with a more specific question, please read the [community guidelines](https://github.com/orgs/bonsai-rx/discussions/864) and create a new discussion for your question.
